System/HIDAPI.hsc

```diff
@@ -295,7 +295,7 @@ sendFeatureReport dev r d = do
   checkWithHidError (n' /= -1) dev "Write failed" "hid_send_feature_report returned -1"
   return $ fromIntegral n'
 
-foreign import ccall unsafe "hidapi/hidapi.h hid_get_serial_number_string"
+foreign import ccall safe "hidapi/hidapi.h hid_get_serial_number_string"
```

Comment:
Despite sounding benign, this one also does udev calls, so `safe` is justified here:

https://github.com/libusb/hidapi/blob/2551a5d84baca323ca8d61db06464cd1b963c289/linux/hid.c#L490-L513